examples/nginx/components/app/nginx.jsonnet

```diff
@@ -0,0 +1,54 @@
+local kap = import "lib/kapitan.libjsonnet";
+local inv = kap.inventory();
+
+local myContainers = [
+    {
+        name: "nginx",
+        image: inv.parameters.nginx.image,
+        ports: [{ containerPort: 80 }],
+    },
+];
+
+local myDeployment = {
+    apiVersion: "apps/v1beta1",
+    kind: "Deployment",
+    metadata: {
+        name: "my-app",
+        namespace: inv.parameters.namespace,
+    },
+    spec: {
+        replicas: inv.parameters.myapp.replicas,
+        template: {
+            metadata: {
+                labels: { app: "my-app" },
+            },
+            spec: {
+                containers: myContainers,
+            },
+        },
+    },
+};
+
+local svc = {
```

Comment:
Can you also split `Deployment` and `Service` into separate files

Reply:
This is still left todo. Just move the `local svc = {... }` to separate files and import them in nginx.jsonnet with `local deployment = import "deployment.jsonnet";` and then the 
```
{
    "app-service": svc,
    "app-deployment": deployment,
}
``` 
stays the same :)

Reply:
service has field which requires `deployment.spec.template`, so if I move svc to other file it will create circular dependency instead i can move deployment to other file and import it in main.jsonnet

Reply:
It's not going to create a circular dependency. Just split deployment and service into separate files, and import in main. You can also import the deployment in the service and reference it, jsonnet can figure it out.